src/SearchRoom/SearchRoom.js

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import './SearchRoom.css';
+import Btn from '@bbc/igm-btn';
+
+function onClick() {
+    var element = document.getElementsByClassName("css-xp4uvy select__single-value")[0].innerHTML;
+    console.log(element);
+    return element;
+}
+
+
+class ViewRoom extends Component {
+    constructor(props) {
+        super();
+        this.toggleBack = this.toggleBack.bind(this);
+        this.state = { DisplayLogin: props.DisplayLogin };
+    }
+
+    toggleBack() {
+        this.setState({ DisplayLogin: !this.state.DisplayLogin });
+    }
+
+    render() {
+        return (
+            <div>
+                <div className="search-room-title">
+                    Search Room
+                </div>
+                <div className="search-room-contents">
+                    <div className="search-room-contents-container">
+                        <div className="textbox">
+                            <div className="label">
+                                <p>Room Name:</p>
+                            </div>
+                            <textarea className="textarea" rows="2">
```

Comment:
General comment, as I know `<textarea>` is used elsewhere, but would it be better for us to use `<input type="text>`?
It feels a bit weird that we can type over multiple lines when entering data such as a Room Name, but maybe this is something worth discussing when we actually implement the form functionality rather than now...

Reply:
I get this point though yes I think we should have a wider discussion about it in the future as its easy to change